libs/providers/go-feature-flag-web/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@openfeature/go-feature-flag-web-provider",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  },
+  "peerDependencies": {
+    "@openfeature/web-sdk": "*"
```

Comment:
@thomaspoignant we could consider now releasing a non-experimental web-sdk version now that the spec is merged, which would allow you to actually version this more appropriately.

Reply:
@toddbaert Yes, it would be even better if we have a final release.

Reply:
I think we can release a sub 1.0 web-sdk tomorrow if you'd like, so you can actually put a proper version here. cc @beeme1mr @lukas-reining @kinyoklion

Reply:
@toddbaert That would be really good.

Reply:
@toddbaert Perfect, I will wait for the release anyway before releasing the provider.

Reply:
https://github.com/open-feature/js-sdk/pull/506#issuecomment-1652158329